_plugins/jekyll_get.rb

```diff
@@ -0,0 +1,39 @@
+require 'json'
+require 'hash-joiner'
+require 'open-uri'
+
+module Jekyll_Get
+  class Generator < Jekyll::Generator
+    safe true
+    priority :highest
+
+    def generate(site)
+      config = site.config['jekyll_get']
+      if !config
+        return
+      end
+      if !config.kind_of?(Array)
+        config = [config]
+      end
+      config.each do |d|
+        begin
+          target = site.data[d['data']]
+          source = JSON.load(open(d['json']))
+          if target
+            HashJoiner.deep_merge target, source
```

Comment:
`_data/cached` is non-standard though. I'd rather not introduce new conventions. We can keep this simple. The data that this would cache accounts for merging remote data with local data (but only if you choose to cache).

Since caching is optional to begin with, I think this behavior is fine.